tags.go

```diff
@@ -77,13 +77,10 @@ func (client *Client) GetTaggableResource(resourceType TaggableResource, identif
 	case TaggableResourceUser:
 		taggableResource, err = client.GetUser(identifier)
 	default:
-		return nil, fmt.Errorf("not a taggable resource type: %s", string(resourceType))
+		err = fmt.Errorf("not a taggable resource type: %s", resourceType)
```

Comment:
This just cleans up the logic a bit - i used the same pattern in GetAliasableResource